src/sql/src/session/vars.rs

```diff
@@ -1725,6 +1733,7 @@ impl SessionVars {
             )
             .with_var(&MAX_QUERY_RESULT_SIZE)
             .with_var(&MAX_IDENTIFIER_LENGTH)
+            .with_var(&MAX_COPY_FROM_SIZE)
```

Comment:
Discussed in person, but writing here for the record, I think we should probably make this a system var like `MAX_RESULT_SIZE`. That way a single session can't decide to OOM environmentd.

Reply:
Good call, updated to make `max_copy_from_size` a SystemVar